packages/pymedphys_pinnacle/src/pymedphys_pinnacle/export/rtplan.py

```diff
@@ -0,0 +1,548 @@
+    # ds.ToleranceTableSequence = Sequence() #figure out where to get this
+    # information
+    ds.FractionGroupSequence = Sequence()
+    ds.BeamSequence = Sequence()
+    ds.PatientSetupSequence = Sequence()  # need one per beam
+    ds.ReferencedStructureSetSequence = Sequence()
+    ReferencedStructureSet1 = Dataset()
+    ds.ReferencedStructureSetSequence.append(ReferencedStructureSet1)
+    ds.ReferencedStructureSetSequence[0].ReferencedSOPClassUID = RTStructSOPClassUID
+    ds.ReferencedStructureSetSequence[0].ReferencedSOPInstanceUID = plan.struct_inst_uid
+    ds.ApprovalStatus = 'UNAPPROVED'  # find out where to get this information
+
+    ds.FractionGroupSequence.append(Dataset())
+    ds.FractionGroupSequence[0].ReferencedBeamSequence = Sequence()
+
+    metersetweight = ['0']
+
+    num_fractions = 0
+    beam_count = 0
+    beam_list = trial_info['BeamList'] if trial_info['BeamList'] else []
+    if len(beam_list) == 0:
+        plan.logger.warning("No Beams found in Trial. Unable to generate RTPLAN.")
+        return None
+    for beam in beam_list:
+
+        beam_count = beam_count + 1
+
+        plan.logger.info("Exporting Plan for beam: " + beam['Name'])
+
+        ds.PatientSetupSequence.append(Dataset())
+        ds.PatientSetupSequence[beam_count - 1].PatientPosition = patient_position
+        ds.PatientSetupSequence[beam_count - 1].PatientSetupNumber = beam_count
+
+        ds.FractionGroupSequence[0].ReferencedBeamSequence.append(Dataset())
+        ds.FractionGroupSequence[0].ReferencedBeamSequence[beam_count - 1].ReferencedBeamNumber = beam_count
+        ds.BeamSequence.append(Dataset())
+        # figure out what to put here
+        ds.BeamSequence[beam_count - 1].Manufacturer = Manufacturer
+        ds.BeamSequence[beam_count - 1].BeamNumber = beam_count
+        ds.BeamSequence[beam_count - 1].TreatmentDeliveryType = 'TREATMENT'
+        ds.BeamSequence[beam_count - 1].ReferencedPatientSetupNumber = beam_count
+        ds.BeamSequence[beam_count - 1].SourceAxisDistance = '1000'
+        ds.BeamSequence[beam_count - 1].FinalCumulativeMetersetWeight = '1'
+        ds.BeamSequence[beam_count - 1].PrimaryDosimeterUnit = 'MU'
+        ds.BeamSequence[beam_count - 1].PrimaryFluenceModeSequence = Sequence()
+        ds.BeamSequence[beam_count - 1].PrimaryFluenceModeSequence.append(Dataset())
+        ds.BeamSequence[beam_count - 1].PrimaryFluenceModeSequence[0].FluenceMode = 'STANDARD'
+
+        ds.BeamSequence[beam_count - 1].BeamName = beam['FieldID']
+        ds.BeamSequence[beam_count - 1].BeamDescription = beam['Name']
+
+
+        if 'Photons' in beam['Modality']:
+            ds.BeamSequence[beam_count - 1].RadiationType = 'PHOTON'
+        elif 'Electrons' in beam['Modality']:
+            ds.BeamSequence[beam_count - 1].RadiationType = 'ELECTRON'
+        else:
+            ds.BeamSequence[beam_count - 1].RadiationType = ""
```

Comment:
I would rather raise an exception here, if its a modality we are not sure about we should noisily tell the user that we are unsure about what to do.

Reply:
In any case where there is any chance the code might be doing "the wrong thing" let's raise an error. I want this to fail fast and noisily if it is ever sees something that is unexpected.